packages/create-atomic/index.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+const path = require('path');
+const args = process.argv.slice(2);
+const {Plop, run} = require('plop');
+const argv = require('minimist')(args);
```

Comment:
I don't see `minimist` in the package.json... Is that expected?

Reply:
Weird right, that's what Plop documents, it's once or their dependencies https://github.com/plopjs/plop/blob/master/package.json#L51

I'll add it just to be safe